sql-cli/sql_cli/_cli.py

```diff
@@ -0,0 +1,61 @@
+from __future__ import annotations
+
+from pathlib import Path
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from airflow.models import DAG
+
+from rich import print as rprint
+from typer import Exit
+
+from sql_cli import dag_generator, run_dag as dag_runner
+from sql_cli.exceptions import ConnectionFailed, DagCycle, EmptyDag, SqlFilesDirectoryNotFound
+from sql_cli.project import Project
+
+
+def _generate_dag(project: Project, env: str, workflow_name: str) -> Path:
```

Comment:
As we discussed, shall we remove the module & methods "_" prefixes? We can refactor tests in a future PR